packages/zowe-explorer/__tests__/__e2e__/step_definitions/views/JobsTableView.steps.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { Then } from "@cucumber/cucumber";
+import { ContextMenu } from "wdio-vscode-service";
+
+Then('the user can right-click on the jobs profile and select "Show as Table"', async function () {
+    const ctxMenu: ContextMenu = await this.profileNode.openContextMenu();
+    await ctxMenu.wait();
```

Comment:
This test times out on MacOS because context menus are native so the selector doesn't work.

Not sure if there's any any solution for this since the relevant WebDriver.io issue is still open: https://github.com/webdriverio-community/wdio-vscode-service/issues/57

We may need to skip this test on MacOS for now if it's unsupported? Or perhaps on MacOS we can run the VS Code command directly and skip right-clicking the node.

Reply:
Resolved in a3c7eec